### PR TITLE
Html2po: option for single file output

### PR DESCRIPTION
**Pull request by anderskaplan**
Sometimes it can be more convenient to work with a single pot/po file for an entire web site, or part of a web site, instead of one per html file. For example, this is a workflow familiar to authors of WordPress plugins. This PR adds that option to html2po.

The function is invoked through the "multifile" command-line option, an option which is provided by some of the other converters for similar purposes. It might be better to create a new option for it. Please let me know what you think.

The PR also removes an obsolete and non-functional option to include untagged content from html2po.

### CODE REVIEW

File: translate/misc/optrecurse.py [hunk 1]
```diff
@@ -102,7 +102,8 @@ def format_option_strings(self, option):
 
 
 class StdoutWrapper:
-    out = sys.stdout
```
Comment (anderskaplan) [inline translate/misc/optrecurse.py:105]: This is to be able to capture the output from a unit test. Without the constructor, the assignment is done too early and no output is captured.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Comment by alexsegura** [comment]
@nijel would it be possible to make a new release to have this option, please? 🙏

**Comment by nijel** [comment]
Yes, I plan to do it soon, but first I'd like to merge some more PRs like https://github.com/translate/translate/pull/4188 or https://github.com/translate/translate/pull/4206

**Pull request merged**